src/lib/1pwd-vault.ts

```diff
@@ -171,24 +170,21 @@ export async function replaceVaultItemFields(
 
 export async function updateVaultItem(
   id: string,
-  newData: {op: "replace" | "add" | "remove", path: string, value: any}[],
-  type?: "apiKey" | "privateKey",
+  newData: { op: "replace" | "add" | "remove", path: string, value: any }[],
+  type?: VaultItemType
 ) {
-  const vaultId =
-    type === "privateKey"
-      ? process.env.OP_PRIVATE_KEY_VAULT_ID!
-      : process.env.OP_API_TOKEN_VAULT_ID!;
+  const vaultId = idPerType[type];
   try {
     const updatedItem = await fetch(
       `${vaultApiUrl}/v1/vaults/${vaultId}/items/${id}`,
       {
         method: "PATCH",
         headers: {
           Authorization: `Bearer ${vaultToken}`,
-          "Content-Type": "application/json",
+          "Content-Type": "application/json"
```

Comment:
youre using those headers in exact same form in 3 palces. pls unify if by making it a const